qa/suites/rbd/nvmeof/base/install.yaml

```diff
@@ -0,0 +1,32 @@
+use_shaman: True
+tasks:
+- cephadm:
+- cephadm.shell:
+    host.a:
+    # get state before nvmeof deployment
+    - ceph orch status
+    - ceph orch ps
+    - ceph orch ls
+    - ceph orch host ls
+    - ceph orch device ls
+    - ceph osd lspools
+    # create pool
+    - ceph osd pool create mypool
+    - ceph osd pool application enable mypool rbd
+    - ceph osd lspools
+    - ceph osd pool ls detail
+    # deploy nvmeof
+    - ceph config set mgr mgr/cephadm/container_image_nvmeof quay.io/ceph/nvmeof:v0.0.5
+    - ceph config get mgr mgr/cephadm/container_image_nvmeof
+    - ceph orch apply nvmeof mypool --placement="1 $(hostname)"
```

Comment:
@caroav , currently the ceph applies only to a single host, the next step should be adding more hosts.

Reply:
Agree but let's deliver this PR like this and then improve. Can you we use mgr/cephadm/container_image_nvmeof quay.io/ceph/nvmeof:**latest** ? Will this work?